applications/nrf_desktop/VERSION

```diff
@@ -3,3 +3,12 @@ VERSION_MINOR = 7
 PATCHLEVEL = 99
 VERSION_TWEAK = 0
 EXTRAVERSION =
+
+APP_ROOT_SEQ_NUM = 1
```

Comment:
Can we consider automatically generating the newly added properties from the basic set:
```
VERSION_MAJOR = 2
VERSION_MINOR = 7
PATCHLEVEL = 99
VERSION_TWEAK = 0
EXTRAVERSION =
```
That may allow for relying on simplified approach for an app that uses a single version and sequence number for DFU (also version could be defined in the same simple way as e.g. for MCUboot). Modifying all of these values could still be possible if user needs it (more advanced DFU scenarios).

Just an potential improvement (to simplify our configuration), not a hard requirement.

Reply:
Added a separate commit with this feature and doc update.